### DISCUSSION TITLE
connect to wifi using command

### DISCUSSION THREAD

**Discussion by dineshu07** [doronz88/pymobiledevice3#830] [Ideas]
Hi All,

I have a requirement that involves connecting the device to WiFi. Is it possible to achieve this through reverse engineering? I'm envisioning something similar to the following:

`pymobiledevice3 developer connect-wifi SSID PASSWORD
`
Is this feasible? I'm using Appium to achieve this. Please let me know if there are any possibilities. I'd like to invest my time and contribute if it's feasible.

**Comment by doronz88** [comment]
From my knowledge, there is no such exported api to achieve this
